Components/Card/card.css

```diff
@@ -86,22 +87,22 @@
   display: flex;
 }
 
-.card-text-overlay{
-    position: absolute;
-    bottom: 0.2rem;
-    left: 0.2rem;
-    color: white;
+.card-text-overlay {
+  position: absolute;
+  bottom: 0.2rem;
+  left: 0.2rem;
+  color: white;
```

Comment:
If possible, include this color as a variable in root, and then set it here so that the code looks consistent.